psi4/driver/driver.py

```diff
@@ -1282,7 +1283,7 @@ def hessian(name, **kwargs):
             """hessian() will perform frequency computation by finite difference of analytic gradients.\n""")
 
         # Obtain list of displacements
-        findif_meta_dict = driver_findif.hessian_from_gradient_geometries(molecule, irrep)
```

Comment:
Move this to `hessian_from_gradient_plan`? Some sort of canonical language should be moved to here:
```
plan
execute
assemble
```

Reply:
I think the `FindifComputer` does have that structure. deferring to final #1351 review